app/views/templates/cover_letter_template.scala.html

```diff
@@ -496,40 +490,97 @@ <h2 class="heading-medium margin-top-small no-margin-bottom print-header__headin
-                    HMRC is not able to give a 3 month extension for this appeal, it can only be given by the independent tribunal.
+                    @messages("pdf.ruling.section.askForReview.requestReview")
                 </p>
-                <p class="govuk-body-s mt-40">
-                    Yours sincerely,
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.askForReview.solsTeam")
+                </p>
+                <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.1")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.2")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.3")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.4")</li>
+                    <li>@messages("pdf.ruling.section.askForReview.solsTeam.5")</li>
+                </ul>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.askForReview.solsTeam.cannotWrite")
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText"))
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.application.footer.solicitors.office.and.legal.services.address"))
+                </p>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealForIndependentTribunal.title")</h4>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.appealImmediately")
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals")
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements")
+                </p>
+                <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                    <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1")</li>
+                    <li>@messages("pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2")</li>
+                </ul>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.relevantInfo")
+                </p>
+                <p class="govuk-body-s">
+                    @messages("pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo")
+                </p>
+                <p class="govuk-body-s">
+                    @Html(messages("pdf.ruling.section.appealForIndependentTribunal.page"))
+                </p>
+                <h4 class="govuk-heading-s">@messages("pdf.ruling.section.appealsAndReviews.title")</h4>
```

Comment:
`govuk-heading-s` makes the h4s a bit smaller than the paragraphs which is why I think the `heading-small` styling was created in this file. Please use the class `heading-small` instead, thanks